dask/array/linalg.py

```diff
@@ -1104,7 +1104,8 @@ def lstsq(a, b):
                            (np.dot, (rt.name, 0, 0), (r.name, 0, 0)))))}
     graph = HighLevelGraph.from_collections(sname, sdsk, dependencies=[rt])
     _, _, _, ss = np.linalg.lstsq(np.array([[1, 0], [1, 2]], dtype=a.dtype),
-                                  np.array([0, 1], dtype=b.dtype))
+                                  np.array([0, 1], dtype=b.dtype),
+                                  rcond=-1)
```

Comment:
@jakirkham do you know if it makes sense to add an `rcond` parameter to `da.linalg.lstsq`?

Reply:
Possibly? I haven't really given this a lot of thought. Should we raise an issue to solicit feedback from others that may care/have thoughts on this?

Reply:
Let's just wait until we have a request.